src/jit-diff/jit-diff.cs

```diff
@@ -547,50 +563,100 @@ public int List()
             "System.Linq.Expressions.dll",
+            "System.Linq.dll",
             "System.Linq.Parallel.dll",
+            "System.Linq.Queryable.dll",
             "System.Net.Http.dll",
             "System.Net.NameResolution.dll",
             "System.Net.Primitives.dll",
             "System.Net.Requests.dll",
             "System.Net.Security.dll",
             "System.Net.Sockets.dll",
+            "System.Net.WebHeaderCollection.dll",
             "System.Numerics.Vectors.dll",
-            "System.Reflection.dll",
+            "System.ObjectModel.dll",
+            "System.Private.Uri.dll",
             "System.Reflection.DispatchProxy.dll",
             "System.Reflection.Emit.ILGeneration.dll",
             "System.Reflection.Emit.Lightweight.dll",
             "System.Reflection.Emit.dll",
             "System.Reflection.Extensions.dll",
             "System.Reflection.Metadata.dll",
+            "System.Reflection.dll",
             "System.Reflection.Primitives.dll",
             "System.Reflection.TypeExtensions.dll",
-            "System.Text.Encoding.dll",
+            "System.Resources.Reader.dll",
+            "System.Resources.ResourceManager.dll",
+            "System.Runtime.Extensions.dll",
+            "System.Runtime.Handles.dll",
+            "System.Runtime.InteropServices.dll",
+            "System.Runtime.InteropServices.RuntimeInformation.dll",
+            "System.Runtime.Loader.dll",
+            "System.Runtime.dll",
+            "System.Runtime.Numerics.dll",
+            "System.Security.Claims.dll",
+            "System.Security.Cryptography.Algorithms.dll",
+            "System.Security.Cryptography.Cng.dll",
+            "System.Security.Cryptography.Csp.dll",
+            "System.Security.Cryptography.Encoding.dll",
+            "System.Security.Cryptography.OpenSsl.dll",
+            "System.Security.Cryptography.Primitives.dll",
+            "System.Security.Cryptography.X509Certificates.dll",
+            "System.Security.Principal.dll",
+            "System.Security.Principal.Windows.dll",
+            "System.Text.Encoding.CodePages.dll",
             "System.Text.Encoding.Extensions.dll",
+            "System.Text.Encoding.dll",
             "System.Text.RegularExpressions.dll",
+            "System.Threading.dll",
+            "System.Threading.Overlapped.dll",
+            "System.Threading.Tasks.Dataflow.dll",
+            "System.Threading.Tasks.Extensions.dll",
+            "System.Threading.Tasks.dll",
+            "System.Threading.Tasks.Parallel.dll",
+            "System.Threading.Thread.dll",
+            "System.Threading.ThreadPool.dll",
+            "System.Threading.Timer.dll",
             "System.Xml.ReaderWriter.dll",
             "System.Xml.XDocument.dll",
-            "System.Xml.XmlDocument.dll"
+            "System.Xml.XmlDocument.dll",
+            "System.Xml.XPath.dll",
+            "System.Xml.XPath.XDocument.dll"
         };
```

Comment:
Good question. The following assemblies don't have any dasm output:

mscorlib.dasm
System.AppContext.dasm
System.Globalization.Calendars.dasm
System.Globalization.dasm
System.IO.FileSystem.Primitives.dasm
System.Reflection.dasm
System.Reflection.Emit.dasm
System.Reflection.Emit.ILGeneration.dasm
System.Reflection.Emit.Lightweight.dasm
System.Reflection.Primitives.dasm
System.Resources.ResourceManager.dasm
System.Runtime.Loader.dasm
System.Security.Principal.dasm
System.Text.Encoding.dasm
System.Threading.Thread.dasm
System.Threading.ThreadPool.dasm
System.Threading.Timer.dasm

I suppose I'll remove them from the list.